src/store/configure.store.ts

```diff
@@ -1,19 +1,29 @@
 type ConfigureState = {
   appName: string
+  access: {
+    selectedChain: string
+  }
 }
 
 const state: ConfigureState = {
   appName: '',
+  access: {
+    selectedChain: '',
```

Comment:
Should we set the default value to Ethereum?

Reply:
Yeah, I guess any one should be default